app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -49,7 +50,8 @@ class PatientsScreenController @Inject constructor(
     private val userClock: UserClock,
     @Named("approval_status_changed_at") private val approvalStatusUpdatedAtPref: Preference<Instant>,
     @Named("approved_status_dismissed") private val hasUserDismissedApprovedStatusPref: Preference<Boolean>,
-    @Named("app_update_last_shown_at") private val appUpdateDialogShownAtPref: Preference<Instant>
+    @Named("app_update_last_shown_at") private val appUpdateDialogShownAtPref: Preference<Instant>,
+    private val illustrationDao: HomescreenIllustration.RoomDao
```

Comment:
Can you move this property up so that it is grouped with the rest of the non-`@Named` properties, please?

Reply:
done